File: Frontend/GWT/gwt-modular-dynamicform/src/com/nonagon/modular/dynamicform/client/FormStatus.java
package com.nonagon.modular.dynamicform.client;

public enum FormStatus {

	Active,
	Inactive,
	Deleted;
	
	public static FormStatus fromString(String value) {

		return Enum.valueOf(FormStatus.class, value);
	}
	
	public static String fromEnum(FormStatus formStatus) {
		
		return formStatus.toString();
	}
}

File: Backend/Mono/Nonagon.Modular.DynamicForm/FormStatus.cs
namespace Nonagon.Modular.DynamicForm
{
	/// <summary>
	/// Form related status.
	/// </summary>
	public enum FormStatus
	{
		/// <summary>
		/// The data is active for use.
		/// </summary>
		Active = 1,

		/// <summary>
		/// The data is not active for use.
		/// </summary>
		Inactive = 0,

		/// <summary>
		/// The data has been deleted.
		/// </summary>
		Deleted = -1
	}
}

